resources/environment.go

```diff
@@ -146,7 +146,7 @@ func UpdateEnvironmentService(db *gorm.DB, client kube.APIInterface, environment
 // GetEnvironment will get an existing Environment by id
 func GetEnvironment(db *gorm.DB, id string) (*Environment, error) {
 	environment := new(Environment)
-	if err := db.Preload("Application").Preload("Versions").Where("uuid = ?", id).First(&environment).Error; err != nil {
+	if err := db.Preload("Application.Environments").Preload("Versions").Where("uuid = ?", id).First(&environment).Error; err != nil {
```

Comment:
This is in order to show the available environments to deploy to on the version details page. Not ideal but it works.